graylog2-server/src/main/java/org/graylog2/filters/StaticFieldFilter.java

```diff
@@ -43,23 +48,32 @@
 
     private static final String NAME = "Static field appender";
 
-    private final Cache<String, List<Map.Entry<String, String>>> cache = CacheBuilder.newBuilder()
-            .expireAfterWrite(1, TimeUnit.SECONDS)
-            .build();
+    private static final List<Map.Entry<String, String>> EMPTY = ImmutableList.of();
```

Comment:
`ImmutableList.of()` already returns a constant. No need to define another one for redirection. 😉 
Alternatively use `Collections.emptyList()` if you want to make the intention totally clear.